custom/data.ts

```diff
@@ -1,73 +1,72 @@
-import { faker } from "@faker-js/faker";
-
 const introduction = {
-  title: "YOUR NAME",
-  text: "And a brief description of you",
+  title: "RAIA",
+  text: "Software Developer • Not a Robot",
 };
 
 const description = {
-  text: faker.lorem.paragraph(),
+  text: `Bringing exploratory mindset to technical leadership,
+        with six years of experience in software development and computational research`,
 };
 
 const projects = [
   {
-    title: faker.lorem.words(2),
-    text: faker.lorem.words(5),
+    title: "Software development",
+    text: "Full stack development description",
     works: [
       {
-        title: faker.word.noun(),
-        text: faker.lorem.sentence(),
-        link: "/",
+        title: "Portfolio",
+        text: "A portfolio project to show my skills and existing projects",
+        link: "https://github.com/raiaqt/betterai-portfolio",
         year: "2024",
         github: true,
         image: "placeholder.png",
       },
     ],
   },
   {
-    title: faker.lorem.words(3),
-    text: faker.lorem.words(4),
+    title: "Competitive programming",
+    text: "Competitive programming description",
     works: [
       {
-        title: faker.word.noun(),
-        text: faker.lorem.sentence(),
-        link: "/",
-        year: "2024",
+        title: "Kattis submissions",
+        text: "Solutions to algorithm and data structure problems on ",
+        year: "2023",
+        link: "https://github.com/raiaqt/kattis-submissions",
         github: true,
         image: "placeholder.png",
       },
       {
-        title: faker.word.noun(),
-        text: faker.lorem.sentence(),
-        link: "/",
-        year: "2024",
+        title: "OnlineJudge submissions",
+        text: "Solutions to algorithm and data structure problems on ",
+        year: "2023",
+        link: "https://github.com/raiaqt/onlinejudge-submissions",
         github: true,
       },
     ],
   },
   {
```

Comment:
Consider providing a more descriptive title and subtitle for the current work section. Update the text to be more specific about the responsibilities and achievements. [View Reference](https://www.linkedin.com/in/raia-quitoriano/)